opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -231,7 +231,8 @@ def create_file(filename: str) -> None:
     """
     global CURRENT_FILE, CURRENT_LINE
     if os.path.exists(filename):
-        raise FileExistsError(f"File '{filename}' already exists.")
+        open_file(filename)
+        raise FileExistsError(f"File '{filename}' already exists. Opened the existing file.")
```

Comment:
> However, this scenario could also happen simply because the agent doesn't know there's another file with the same name. E.g. it might attempt to create a `tmp` file but a `tmp` file already exists. By simply raising `FileExistsError`, LLM could decide what to do (e.g. use a different file name).

@tobitege or it may forget.